tests/gold_tests/transaction/txn.test.py

```diff
@@ -33,7 +33,7 @@
 server = Test.MakeOriginServer("server")
 
 Test.testName = ""
-request_header = {"headers": "GET / HTTP/1.1\r\n\r\n",
+request_header = {"headers": "GET / HTTP/1.1\r\nHost: txn.test\r\n\r\n",
```

Comment:
Missing host headers are [rejected](https://github.com/apache/trafficserver/blob/8d01164257417434b667021365b5f07d0ec596e0/tests/tools/sessionvalidation/sessionvalidation.py#L195-L198) when the microserver loads its sessions, and so the health check fails unless this is corrected.